openmc/deplete/abc.py

```diff
@@ -819,9 +819,31 @@ def _get_bos_data_from_restart(self, source_rate, bos_conc):
         return bos_conc, OperatorResult(k, rates)
 
     def _get_start_data(self):
+        """
+        This function fetches the starting state of a simulation in terms of the
+        simulation physical time at which to start and the index at which the
+        depletion simulation should start at. When no previous results exist,
+        the time and index are both zero. When previous results do exist, it 
+        returns the time corresponding to beginning the previous resutls last
+        timestep and the index as N-1 where N is the number of previous StepResults
+        found in the previous Results.
```

Comment:
I feel a need to clarify that this N-1 index is the obvious result for a file with N results
```suggestion
        found in the previous Results. (as expected from 0-based indexing)
```